.github/workflows/push.yml

```diff
@@ -90,10 +90,17 @@ jobs:
           restore-keys: clippy-${{ runner.os }}-${{ hashFiles('rust-toolchain.toml') }}
 
       - name: Install utils for caching
+        if: runner.os != 'Windows'
         run: |
           test -x ~/.cargo/bin/cargo-sweep || cargo install cargo-sweep
           test -x ~/.cargo/bin/cargo-cache || cargo install cargo-cache --no-default-features --features ci-autoclean cargo-cache
```

Comment:
I'm not sure this test is necessary in the first place, repeated installation should make no difference:
```
nazar-pc@nazar-pc:~> cargo install cargo-outdated
    Updating crates.io index
     Ignored package `cargo-outdated v0.13.1` is already installed, use --force to override
```

Reply:
Removed the test from clippy job